src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -85,6 +86,27 @@ const GuideStore = Reflux.createStore({
     this.updateCurrentGuide();
   },
 
+  recordCue(id, cue) {
+    HookStore.get('analytics:event').forEach(cb =>
+      cb('assistant.guide_cued', {
+        guide: id,
+        cue,
+      })
+    );
+  },
+
+  updatePrevGuide(bestGuide) {
+    if (!bestGuide) return;
+
+    if (
+      !this.state.prevGuide ||
+      (this.state.prevGuide && this.state.prevGuide.id !== bestGuide.id)
```

Comment:
i think you can actually safely drop `this.state.prevGuide && ` because it will never evaluate the second expression if the first is falsey